samples/net/google_iot_device/src/config.c.in

```diff
@@ -0,0 +1,20 @@
+#include "iotc.h"
+
+/* CHANGE THIS */
+const char* iotc_project_id = "macro-precinct-211108";
+/* CHANGE THIS */
+const char* iotc_device_path = "projects/macro-precinct-211108/locations/us-central1/registries/agross-registry/devices/pfalcon-test-device";
+/* CHANGE THIS */
+const char* iotc_publish_topic = "/devices/pfalcon-test-device/state";
+/* CHANGE THIS*/
+const char* iotc_private_key_pem =
+"-----BEGIN EC PRIVATE KEY-----\n"
+"use\n"
+"your\n"
+"key\n"
+"-----END EC PRIVATE KEY-----\n";
+
+const char* iotc_publish_message = "Hello from Zephyr!";
+const char* iotc_private_key_filename = "(none)";
```

Comment:
`iotc_private_key_filename` won't be required then. This line can be deleted.

Reply:
> iotc_private_key_filename won't be required then. This line can be deleted.

I put it because it was referenced from somewhere (error message or something). Now that you ok'ed removing unneeded code, I'll make sure to delete it too.